### PR TITLE
Issue8 seperate lint into frontend and backend folders

### PR DESCRIPTION
**Pull request by hondahmd**
- the linter in the 'whole' root dir has been deleted.
- now backend and frontend have their own linters.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,17 +20,7 @@
...
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.1.1",
-    "eslint-plugin-react": "^7.16.0",
-    "eslint-plugin-react-hooks": "^1.7.0",
-    "eslint-plugin-react-native": "^3.8.1",
-    "pre-commit": "^1.2.2",
-    "prettier": "^1.19.1"
+  "dependencies": {
+    "pre-commit": "^1.2.2"
```
Comment (adamgyulavari) [inline package.json]: @hondahmd you dont use this dependency anywhere

File: lint.sh [hunk 2]
```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
-FILES_CHANGED=$(git diff --name-only HEAD~ HEAD | grep '.js')
-$(./node_modules/eslint/bin/eslint.js ${FILES_CHANGED} --ext .js --fix)
\ No newline at end of file
+# FILES_CHANGED=$(git diff --name-only HEAD~ HEAD | grep '.js')
+# $(./node_modules/eslint/bin/eslint.js ${FILES_CHANGED} --ext .js --fix)
```
Comment (adamgyulavari) [inline lint.sh]: @hondahmd don't leave outcommented code in pushed code

File: package.json (hunk 1 above)
Comment (hondahmd) [reply package.json]: @adamgyulavari well it's used for lint before commit, and used as [npm script](https://github.com/green-fox-academy/JSA-Opal-ShenZhen/blob/honda-issue8-lint/package.json#L10)

File: lint.sh (hunk 2 above)
Comment (hondahmd) [reply lint.sh]: @adamgyulavari roger that, deletion done

**Pull request merged**